hadoop-yarn-project/hadoop-yarn/hadoop-yarn-ui/pom.xml

```diff
@@ -202,6 +202,13 @@
             <groupId>ro.isdc.wro4j</groupId>
             <artifactId>wro4j-maven-plugin</artifactId>
             <version>1.7.9</version>
+              <dependencies>
+                <dependency>
+                  <groupId>org.mockito</groupId>
+                  <artifactId>mockito-core</artifactId>
+                  <version>2.18.0</version>
+                </dependency>
+              </dependencies>
```

Comment:
Thanks @vbmudalige for the pull request.
* Would you comment that "TODO: remove this dependency after upgrading wro4j-maven-plugin to 1.8.1 or later"?
* Would you remove 2 whitespaces from the indents?

Reply:
@aajisaka, I made the changes. Please review.